src/analysis/processing/qgsrasteranalysisutils.h

```diff
@@ -65,6 +69,21 @@ namespace QgsRasterAnalysisUtils
    * \see createRasterTypeParameter()
    */
   Qgis::DataType rasterTypeChoiceToDataType( int choice );
+
+  struct BooleanLogicInput
```

Comment:
Could we rename this to something more generic, that'll come in handy for another forthcoming algorithm :)

Reply:
sure!